CODE_OF_CONDUCT.md

```diff
@@ -1 +1,89 @@
-TBD
+# FaithTech LABS Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+FaithTech LABS pledge to making participation in
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance, race, religion, or sexual identity and
```

Comment:
I believe we can be strong in our faith and accepting of others. I think we can leave religion in this list. See the Our Faith section below.